packages/mask/src/social-network-adaptor/twitter.com/injection/PostActions/selector.ts

```diff
@@ -0,0 +1,7 @@
+import type { LiveSelector } from '@dimensiondev/holoflows-kit'
+import { querySelector } from '../../utils/selector'
+
+type E = HTMLElement
+export const selector = ''
+export const postActionsSelector: () => LiveSelector<E, true> = () =>
```

Comment:
Plz move this to `../../utils/selector`.

Reply:
This file is no longer needed. I originally wanted to put the selector alongside to the usage.